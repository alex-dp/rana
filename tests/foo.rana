foo(bar="baz" fizz) {
	buzz(divisible="5, 3"){
		fifteen(){}
		thirty(){}
		fortyfive(){}
	}

	constantinople(new="rome" rechristener="constantine"){
		current(name="istanbul"){
			country(name="turkey"){}
		}
	}
}