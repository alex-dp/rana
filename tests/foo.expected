<foo bar="baz" fizz> 
	<buzz divisible="5, 3">
		<fifteen></fifteen>
		<thirty></thirty>
		<fortyfive></fortyfive>
	</buzz>

	<constantinople new="rome" rechristener="constantine">
		<current name="istanbul">
			<country name="turkey"></country>
		</current>
	</constantinople>
</foo>